eventlistener/lager_event_listener.go

```diff
@@ -24,14 +24,18 @@ type LagerEventListener struct {
 //Event is a method for Lager event listening
 func (e *LagerEventListener) Event(event *core.Event) {
 	logger := openlogging.GetLogger()
-	logger.Debugf("Get lager event, key: %s, type: %s", event.Key, event.EventType)
+	fmt.Printf("Get lager event, key: %s, type: %s, value: %v", event.Key, event.EventType, event.Value)
```

Comment:
merge 26 and 27 into one line, and from now must use structured log info, refer to my code "openlogging.WithTag"

Reply:
change.